tests/commands/hijack_fd.py

```diff
@@ -0,0 +1,18 @@
+"""
+`hijack_fd` command test module
+"""
+
+
+from tests.utils import GefUnitTestGeneric, gdb_run_cmd
+
+
+class HijackFdCommand(GefUnitTestGeneric):
+    """`hijack-fd` command test module"""
+
+
+    cmd = "hijack-fd"
+
+
+    def test_cmd_hijack_fd(self):
+        res = gdb_run_cmd(f"{self.cmd}")
+        self.assertNoException(res)
```

Comment:
Maybe we can actually check the FDs after to make sure it worked as expected. We aren't providing any args so this does nothing but prints the syntax.

Test for something like this

```
gef➤  hijack-fd 1 /tmp/x
[+] Opened '/tmp/x' as fd #3
[+] Duplicated fd #3 → #1
[+] Closed extra fd #3
[+] Success
gef➤  info inferior
  Num  Description       Executable
* 1    process 35642     /usr/bin/cat
gef➤  !ls -l /proc/35642/fd
total 0
lrwx------ 1 root root 64 Feb 13 15:04 0 -> /dev/pts/1
lrwx------ 1 root root 64 Feb 13 15:04 1 -> /tmp/x
lrwx------ 1 root root 64 Feb 13 15:04 2 -> /dev/pts/1
```